server/id/id.go

```diff
@@ -45,14 +46,24 @@ type allocatorImpl struct {
 	base uint64
 	end  uint64
 
-	client   *clientv3.Client
-	rootPath string
-	member   string
+	client    *clientv3.Client
+	rootPath  string
+	allocPath string
+	label     string
+	metric    prometheus.Gauge
+	member    string
 }
 
 // NewAllocator creates a new ID Allocator.
-func NewAllocator(client *clientv3.Client, rootPath string, member string) Allocator {
-	return &allocatorImpl{client: client, rootPath: rootPath, member: member}
+func NewAllocator(client *clientv3.Client, rootPath, allocPath, label, member string) Allocator {
```

Comment:
The data type of the parameter should be defined here?
For example，```rootPath``` should be ```rootPath string``` ?

Reply:
It can be omitted since they are of the same type. (there is a string at the end)
or is this more of a concern of style?